src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -168,8 +168,7 @@
             {% set label = name|humanize %}
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
-            {{- widget|raw -}}
-            {{- label is not sameas(false) ? label|trans({}, translation_domain) -}}
+            {{- widget|raw }} {{ label is not sameas(false) ? label|trans({}, translation_domain) -}}
```

Comment:
Never mind, this has been fixed in the meantime in 99a1fcc3e2d51a9be176ad5d8ed1fcda4a4492d7.